keyboards/bacca70/keymaps/via/keymap.c

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2022 keebnewb
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[0] = LAYOUT_default(
```

Comment:
This needs a bit of cleaning up, there are mixed tabs and spaces.

Reply:
@fauxpark I've updated the keymaps and changed the spaces to tabs 👍

Reply:
I think I'd prefer spaces, tbh:
![image](https://user-images.githubusercontent.com/4781841/158088487-d4654d91-b988-4386-8ede-a7bb548f7494.png)

Reply:
@fauxpark updated! sorry I switched to a different code editor and that helped 😂